http/http_connection.hpp

```diff
@@ -716,64 +669,54 @@ class Connection :
 
     void cancelDeadlineTimer()
     {
-        if (timerCancelKey)
-        {
-            BMCWEB_LOG_DEBUG << this << " timer cancelled: " << &timerQueue
-                             << ' ' << *timerCancelKey;
-            timerQueue.cancel(*timerCancelKey);
-            timerCancelKey.reset();
-        }
+        timer.cancel();
     }
 
-    void startDeadline(size_t timerIterations)
+    void startDeadline()
     {
         cancelDeadlineTimer();
 
-        if (timerIterations)
+        std::chrono::seconds timeout(120);
```

Comment:
@joseph-reynolds  @anoo1 How do you feel about these numbers? 26 mins for users logged in (like from the GUI). 
2 mins for users not logged in. We could go 26mins/26 mins, but I don't see a use case.